src/main/java/com/clussmanproductions/trafficcontrol/item/ItemCrossingRelayBox.java

```diff
@@ -12,7 +13,11 @@
 import com.clussmanproductions.trafficcontrol.blocks.BlockCrossingRelayTopSW;
 import com.clussmanproductions.trafficcontrol.tileentity.RelayTileEntity;
 
+import net.minecraft.client.Minecraft;
```

Comment:
I still have some concerns about using client-only fields in a common class like this, but as long as it works on a dedicated server, I guess it's fine.

Reply:
Server-side stability not impacted.